tests/unit/Composer/Read/Register/ReadByteAddressSplitterTest.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Tests\unit\Composer\Read\Register;
+
+use ModbusTcpClient\Composer\AddressSplitter;
+use ModbusTcpClient\Composer\Read\Register\ByteReadRegisterAddress;
+use ModbusTcpClient\Composer\Read\Register\ReadRegisterAddressSplitter;
+use PHPUnit\Framework\TestCase;
+
+class ReadByteAddressSplitterTest extends TestCase
+{
+    public function testSplitSameAddress()
+    {
+
+        $splitter = new ReadRegisterAddressSplitter(ByteReadRegisterAddress::class);
```

Comment:
This line should be `$splitter = new ReadRegisterAddressSplitter(ReadHoldingRegistersRequest::class);` to work